app/javascript/packs/containers/OptionsContainer.jsx

```diff
@@ -13,7 +14,43 @@ import { getAnswerForQuestionAndBuilding } from '../selectors/answersSelector';
 import { getDependentQuestionsForOptions } from '../selectors/questionsSelector';
 import { createAnswer, updateAnswer } from '../actions/answers';
 
+const styles = {
+  transition: 'all 0.4s cubic-bezier(1, 0, 0, 1)',
+  overflowY: 'hidden'
+};
+
+const transitionStyles = {
+  entering: {
+    maxHeight: 0,
+    transform: 'translateX(-20px)',
+    marginLeft: 0,
+    opacity: 0
+  },
+  entered: {
+    maxHeight: 800,
+    marginLeft: 20,
+    opacity: 1
+  },
+  exiting: {
+    maxHeight: 800,
+    opacity: 1
+  },
+  exited: {
+    maxHeight: 0,
+    marginLeft: 0,
+    opacity: 0
+  }
+};
+
```

Comment:
jw why are styles defined in the react components vs css?

Reply:
CSS comes in a Webpack import after Sass processing, not a `<link>` reference to bundle output